tests/kafkatest/services/verifiable_producer.py

```diff
@@ -37,7 +39,19 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
 
+        # If security protocol is not specified, use the default in the template properties.
+        self.prop_file = self.render('verifiable_producer.properties')
+        self.security_config = SecurityConfig(security_protocol, self.prop_file)
```

Comment:
As mentioned above, to avoid empty dummy files, we can just do something like this for now:
```
self.prop_file = ""
self.security_config = SecurityConfig(security_protocol, self.prop_file)
self.security_protocol = self.security_config.security_protocol
self.prop_file += str(self.security_config)
```